app/src/main/java/hr/dice/filipbionda/tmdbpractice/ui/theme/Type.kt

```diff
@@ -0,0 +1,33 @@
+package hr.dice.filipbionda.tmdbpractice.ui.theme
+
+import androidx.compose.material3.Typography
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontFamily
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.unit.sp
+
+val Typography =
+    Typography(
+        bodyLarge =
+        TextStyle(
+            fontFamily = FontFamily.Default,
+            fontWeight = FontWeight.Normal,
+            fontSize = 16.sp,
+            lineHeight = 24.sp,
+            letterSpacing = 0.5.sp,
+            color = white,
+        ),
+        bodySmall =
```

Comment:
I know it is a formatting and styling issue, but the code is usually formatted to be inlined to look like this:
```
bodySmall = TextStyle(
    fontFamily = FontFamily.Default,
    fontWeight = FontWeight.Normal,
    fontSize = 16.sp,
    color = white,
),
```
I see that sometimes you write it inlined and sometimes you do not. Try to be consistent with the way you write code. It will be beneficial in the long run because it will be easier to read

Reply:
Done